### PR TITLE
Guidelines for new proposals

### PR DESCRIPTION
**Pull request by dgovil**
### Description of Proposal

This is meta proposal that provides suggestions for putting up new proposals, such that they'll be easier to digest and provide feedback on.

Things like what information a proposal could include, structuring guidance and recommendations to include schema snippets.

This isn't pointed at any specific proposals, but it is feedback we've received internally when trying to collect feedback in the past. It's something I believe would make it easier to get more feedback and higher quality feedback.

[Link to Rendered Proposal](https://github.com/dgovil/USD-proposals/blob/ProposalGuidance/README.md#guidance-for-writing-proposals)

### Contributing

- [X] I agree to and accept the [Supplemental Terms](https://graphics.pixar.com/usd/release/contributing_supplemental.html).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,42 @@ When a proposal has taken enough shape that it warrants detailed feedback and it
...
+   5. **Details** about your proposal, such as why you are making certain choices
+   6. **Risks** that you anticipate (if any)
+   7. **Alternate solutions** that you have considered (if any), and why you didn't go with them
+   8. Any **excluded topics**, that you have left out and why. These may be things you want to handle separately in the
+      future for example.
+
+3. It is highly recommended that any submitted proposal include text examples of what your proposal would look like
+   in `.usda` syntax.
+4. It's recommended that long sentences be split over multiple lines in your Markdown file.
+   This allows more granular feedback, and easier viewing of the files. A recommended line length is 120 characters.
```
Comment (meshula) [inline README.md:59]:
```suggestion
   This allows more granular feedback, and easier viewing of the files.
```
I suggest we don't recommend line lengths. Doesn't seem worth opening a debate about "holy" line lengths which specifying 120 would seem to beg.

**Review by meshula — CHANGES_REQUESTED** [review]
One tiny request then it's good to land.

**Comment by meshula** [comment]
Thanks for this, it'll definitely help people structure proposals and understand what makes a proposal useful.

File: README.md (hunk 1 above)
Comment (dgovil) [reply README.md]: Removed!

**Review by meshula — APPROVED** [review]

**Comment by meshula** [comment]
@dgovil Any idea what's going on with the Unverified tag on your commit? You didn't signoff the commit, so that perhaps could have caused the Unverified, but clicking on it lists reasonable looking certs but says "The signing certificate or its chain could not be verified."

**Comment by dgovil** [comment]
I'm not sure. I just tried re-signing but like you say: it sees that it's signed , it just doesn't see the signature as verified.
It's probably some mismatch between internal and external signing. I'll look into it.

**Pull request merged**